dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/AttributeValidationHook.java

```diff
@@ -103,6 +115,31 @@ else if ( valueType.equals( ValueType.USERNAME ) )
         }
     }
 
+    protected void validateAttributeValue( ValidationErrorReporter reporter, TrackerDto trackerDto,
+        TrackedEntityAttribute tea,
+        String value )
+    {
+        checkNotNull( tea, TRACKED_ENTITY_ATTRIBUTE_VALUE_CANT_BE_NULL );
+        checkNotNull( value, TRACKED_ENTITY_ATTRIBUTE_VALUE_CANT_BE_NULL );
+
+        // Validate value (string) don't exceed the max length
+        reporter.addErrorIf( () -> value.length() > Constant.MAX_ATTR_VALUE_LENGTH, trackerDto,
+            E1077, value,
+            Constant.MAX_ATTR_VALUE_LENGTH );
+
+        // Validate if that encryption is configured properly if someone sets
```

Comment:
Did we do this before? I'm not sure this is the correct place to complain about the encryption configuration issue